front/pages/w/[wId]/builder/assistants/[aId]/index.tsx

```diff
@@ -53,14 +51,11 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
       };
     }
 
-    const [
-      { spaces, dataSourceViews, dustApps, mcpServerViews },
-      configuration,
-    ] = await Promise.all([
-      getAccessibleSourcesAndApps(auth),
-      getAgentConfiguration(auth, context.params?.aId as string, "light"),
-      MCPServerViewResource.ensureAllAutoToolsAreCreated(auth),
```

Comment:
I'm not familiar with this, do I have to create a new endpoint to call this on client side before fetching `mcpServerViews`?